portability-api/src/main/java/org/dataportabilityproject/webapp/StartCopyHandler.java

```diff
@@ -124,13 +124,13 @@ private void handleWorkerAssignmentFlow(HttpExchange exchange, UUID jobId) throw
     Preconditions.checkArgument(
         !Strings.isNullOrEmpty(importAuthCookieValue), "Import auth cookie required");
 
-    // We have the data, now update to 'pending worker assignment' so a worker may be assigned
+    // We have the data, now update to 'CREDS_AVAILABLE' so a worker may poll the job.
```

Comment:
We also have the existing job here and can validate it was INITIAL before setting the new state.

Reply:
Added TODO since this is the LegacyPortabilityJob impl